doc/api/util.md

```diff
@@ -167,6 +167,14 @@ corresponding argument. Supported placeholders are:
 * `%f` - Floating point value.
 * `%j` - JSON.  Replaced with the string `'[Circular]'` if the argument
 contains circular references.
+* `%o` - Object. A string representation of an object
+  with generic JavaScript object formatting.
+  Similar to `util.inspect()` with options `{ showHidden: true, depth: 4, showProxy: true }`.
```

Comment:
Just for fun, I took a stroll down memory lane:
This is where is where `depth` was introduced https://github.com/nodejs/node/commit/a2714be8b5c397700df4c80ba6f4abbaab073ea9
Two hour later the defaults were set https://github.com/nodejs/node/commit/de1521413ef53bd63c87a9698eb3611790bc5a56
Values seem to be a personal preferance https://groups.google.com/d/msg/nodejs/rAYFIeJ_72U/_bWLezvZavEJ